en/admin-guide/data-export/interlisexport.rst

```diff
@@ -0,0 +1,180 @@
+.. _export-interlis:
+
+Export INTERLIS Data
+====================
+
+This represents a guide on how to export data from QGEP to INTERLIS 2 Transfer Format (xtf).
+
+
+General
+^^^^^^^^^^^^^
+
+INTERLIS Export can be done using the ili2pg tool from Eisenhut Informatik and a series of prepared SQL scripts. They will do the following:
+* Translation Englisch – German / French (Classes / Attributes)
+* Conversion of value list to / from numeric values (Integer)
+* Restructuring of data from relational to object oriented modelling
+* Conversion of OID (16 characters) to foreign keys TID (integer)
+
+ .. figure:: images/interlisexport.jpg
+ 
+Step by step
+^^^^^^^^^^^^^
+
+Tool
+--------
+Download the latest version of `ili2pg <http://eisenhutinformatik.ch/interlis/ili2pg>`_
+
+ .. figure:: images/eisenhut.jpg
+
+Generate ili2pg schema with data model
+--------------------------------------
+
+You need a valid INTERLIS 2 Transferdataset, the corresponding INTERLIS model
+
+This example uses the Transferdataset transferdatensatz2015_d_mod.xtf and the VSA-DSS 2015 Modell file with all it's corresponding models.
+
+We create a batch file for the import with ili2pg to create an ili2pg schema (named ``vsa_dss_2015_2_d``) in postgres *importdaten_VSA_DSS_2015_2_d_381.bat*
+
+Adapt for your environment if necessary.
+
+>>> java -jar ili2pg.jar --createEnumTxtCol --import --importTid --sqlEnableNull --createEnumTabs --createFk  --noSmartMapping --dbdatabase qgep --dbschema vsa_dss_2015_2_d --dbusr postgres --dbpwd sjib  --log importdaten_VSA_DSS_2015_2_d_361.log transferdatensatz2015_d_mod.xtf
```

Comment:
Shortly checking back for this. It looks like all the `::` and indent of subsequent code lines are still missing. Let's see what it looks like when the result is ready and create a followup if required.